packages/shared/src/hooks/feed/useSortedFeeds.ts

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from 'react';
+import type { Edge } from '../../graphql/common';
+import type { Feed } from '../../graphql/feed';
+import useCustomDefaultFeed from './useCustomDefaultFeed';
+
+export type UseSortedFeedsProps = {
+  edges?: Edge<Feed>[];
+};
+
+export type UseSortedFeeds = Edge<Feed>[];
+
+export const useSortedFeeds = ({
+  edges,
+}: UseSortedFeedsProps): UseSortedFeeds => {
```

Comment:
Made this as a hook vs API because on API `defaultFeedId` is on user so sorting over that would just be complicated and this sounds like something product will change often so this flow is better.